Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -1242,6 +1242,11 @@ public string Search
     /// </summary>
     [Parameter] public EventCallback<string> SearchChanged { get; set; }
 
+    /// <summary>
+    /// If true, the searched text will be highlighted in the dropdown list items based on <see cref="Search"/> value.
+    /// </summary>
+    [Parameter] public bool HighlightSearch { get; set; }
```

Comment:
If user uses ItemContent, he looses Highlight capabilities right, basically he should just used the Highlighter himself, everything should work ok, right?

Reply:
Yes, that is right. We have no way of knowing what the user is doing in the template. So he needs to highlight manually.